app/client/packages/design-system/widgets/src/components/Select/src/SelectTrigger.tsx

```
@@ -0,0 +1,40 @@
+  isInvalid?: boolean;
+  placeholder?: string;
+}
+
+export const SelectTrigger: React.FC<SelectTriggerProps> = (props) => {
+  const { isInvalid, isLoading, placeholder, size } = props;
+
+  return (
+    <Group className={textInputStyles.inputGroup}>
+      <Button
+        className={textInputStyles.input}
+        data-invalid={Boolean(isInvalid) ? "" : undefined}
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)